linux/tests/threadpool_linux_ut/mock_threadpool.h

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+#ifndef MOCK_THREADPOOL_H
```

Comment:
I don't know if this file is necessary.  Since you have the MOCKABLE_FUNCTION in test file.  You might want to play around with it if you feel like it.

Reply:
removed